gradle/libs.versions.toml

```
@@ -3,20 +3,20 @@ agp = "7.1.2"
 anvil = "2.4.0"
 bugsnagGradle = "7.2.0"
 dependencyAnalysisPlugin = "0.80.0"
-detekt = "1.19.0"
-dokka = "1.6.10"
+detekt = "1.20.0"
+dokka = "1.6.20"
 errorproneGradle = "2.0.2"
 java = "17"
 javapoet = "1.13.0"
-kotlin = "1.6.20"
-ksp = "1.6.20-1.0.4"
-ktfmt = "0.34"
+kotlin = "1.6.21"
+ksp = "1.6.21-1.0.5"
+ktfmt = "0.35"
 mavenPublish = "0.19.0"
 moshi = "1.13.0"
 moshix = "0.17.1"
 nullawayGradle = "1.3.0"
-okhttp = "4.9.3"
-spotless = "6.4.2"
+okhttp = "5.0.0-alpha.6"
```

There was a bug with alpha.6 so we took alpha.7 recently on our main repo

```suggestion
okhttp = "5.0.0-alpha.7"
```